CompetitionTemplate/src/competition/subsystems/turret/rotation/commands/RotateTurretWithGamepadCommand.java

```diff
@@ -5,11 +5,15 @@
 import competition.operator_interface.OperatorInterface;
 import competition.subsystems.turret.rotation.TurretRotationSubsystem;
 import xbot.common.command.BaseCommand;
+import xbot.common.controls.sensors.XJoystick;
 
 public class RotateTurretWithGamepadCommand extends BaseCommand {
 
     final TurretRotationSubsystem rotationSubsystem;
     final OperatorInterface oi;
+    
+    final XJoystick leftTrigger = null;
```

Comment:
Are you _sure_ this is what you want to do? `leftTrigger` and `rightTrigger` will always be `null`.